### DISCUSSION TITLE
Fabrication of PaperQA2

### DISCUSSION THREAD

**Discussion by 403-html** [Future-House/paper-qa#421] [General]
This is rather something to think about and to be put into backlog. I looked at the white paper and the fabrication of answers (in the document titled “Contradictions”) is ~ 3%. That's quite a lot. There are tools (so far I've only found paid ones, but I'm sure there are free ones as well) that combat this, such as Patronus AI. Maybe in future iterations it would be worth looking into LLM's fabrication?

**Comment by SamWright822** [comment]
Thanks for opening this issue! This seems more like a discussion than an issue with the code, so I'm going to convert to a discussion
